cpp/tests/binaryop/binop-integration-test.cpp

```diff
@@ -2200,6 +2200,21 @@ TYPED_TEST(FixedPointTestBothReps, FixedPointBinaryOpMultiplyScalar)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view());
 }
 
+TYPED_TEST(FixedPointTestBothReps, FixedPointBinaryOpSimplePlus)
```

Comment:
Is this just a missing binop test? I can't tell if it's testing something more specific/related to this PR.

Reply:
cc: @codereport; IIRC it's just a test we wrote because this case wasn't working in Python. Eventually, we traced that to a Python bug, but we decided to keep this test because why would you delete a test? :D

Reply:
Any test that ever failed is a good test :+1: 
I asked just to make sure it covers what it's supposed to.